static/docs/commands-reference/remote.md

```diff
@@ -94,7 +97,7 @@ DVC config file would look like:
 2. Add AWS S3 remote and modify its region:
 
 ```dvc
-    $ dvc remote add myremote s3://mybucket/myproject
+    $ dvc remote add -f myremote s3://mybucket/myproject
```

Comment:
let's not use `-f` here, `-f` can be dangerous and folks should explicitly choose to use it. If your concern is that this command fails since `myremote` already exists from the previous run, let's just change the name - here and in the command below

Reply:
Yes, will change the name.